### ISSUE TITLE
micromamba hash mismatch 

### ISSUE THREAD

**Issue by boriskaus** [cjdoris/PythonCall.jl#238]
I'm using PythonCall as part of my package, and am consistently getting the follow bug in the CI (used to work fine).

```Julia
Downloading artifact: micromamba-0.27.0
224
 Downloading artifact: micromamba-0.27.0
[225](https://github.com/JuliaGeodynamics/LaMEM.jl/actions/runs/3338177863/jobs/5525499345#step:6:228)
┌ Error: Hash Mismatch!
[226](https://github.com/JuliaGeodynamics/LaMEM.jl/actions/runs/3338177863/jobs/5525499345#step:6:229)
│   Expected sha256:   4adbf3091a4159af2c48264a8e32ecb98147b0e3f200601f384f8f53a6910ca2
[227](https://github.com/JuliaGeodynamics/LaMEM.jl/actions/runs/3338177863/jobs/5525499345#step:6:230)
│   Calculated sha256: 1b1bdd0798c3c57eda3c76660ed148a59d70e555c6dcd3c099e824bf78873d70
[228](https://github.com/JuliaGeodynamics/LaMEM.jl/actions/runs/3338177863/jobs/5525499345#step:6:231)
└ @ Pkg.PlatformEngines /opt/hostedtoolcache/julia/nightly/x64/share/julia/stdlib/v1.9/Pkg/src/PlatformEngines.jl:624
[229](https://github.com/JuliaGeodynamics/LaMEM.jl/actions/runs/3338177863/jobs/5525499345#step:6:232)
ERROR: LoadError: InitError: Unable to automatically download/install artifact 'micromamba-0.27.0' from sources listed in '/home/runner/.julia/packages/MicroMamba/rCGZ4/Artifacts.toml'.
[230](https://github.com/JuliaGeodynamics/LaMEM.jl/actions/runs/3338177863/jobs/5525499345#step:6:233)
Sources attempted:
[231](https://github.com/JuliaGeodynamics/LaMEM.jl/actions/runs/3338177863/jobs/5525499345#step:6:234)
- https://pkg.julialang.org/artifact/d45b66605cad8bca223a3eb103eebb71adcf1415
[232](https://github.com/JuliaGeodynamics/LaMEM.jl/actions/runs/3338177863/jobs/5525499345#step:6:235)
    Error: RequestError: HTTP/2 404 while requesting https://pkg.julialang.org/artifact/d45b66605cad8bca223a3eb103eebb71adcf1415
[233](https://github.com/JuliaGeodynamics/LaMEM.jl/actions/runs/3338177863/jobs/5525499345#step:6:236)
- https://micro.mamba.pm/api/micromamba/linux-64/0.27.0
[234](https://github.com/JuliaGeodynamics/LaMEM.jl/actions/runs/3338177863/jobs/5525499345#step:6:237)
    Error: Verification failed. Download does not match expected hash
```    
See the full CI [here](https://github.com/JuliaGeodynamics/LaMEM.jl/actions/runs/3338177863/jobs/5525499345).

I'm not sure whether the problem is here or down the line in the MicroMamba.jl package, but can imagine that other users may encounter the same. Help is appreciated.

**Comment by nrontsis** [comment]
I have the same issue

**Comment by boriskaus** [comment]
It seems to me that MicroMamba v0.1.10 is broken.

A workaround is to explicitly install MicroMamba 0.1.9 before installing `PythonCall`:
```Julia
julia> ]
pkg> add MicroMamba@0.1.9
pkg> add PythonCall
```
obviously a better solution would be to fix 0.1.10

**Comment by cjdoris** [comment]
Fixed now thanks.
